libcodechecker/server/client_db_access_server.py

```diff
@@ -120,10 +125,13 @@ def check_auth_in_request(self):
         return success
 
     def do_GET(self):
-        authToken = self.check_auth_in_request()
-        if authToken:
+        authSession = self.check_auth_in_request()
```

Comment:
Also a variable named `authSession` is not very Pythonic. This is my bad when I wrote the original version of this code I didn't really know the naming conventions. :innocent: Python uses `auth_session`.